alws/utils/pulp_client.py

```diff
@@ -160,6 +160,21 @@ async def get_rpm_remote(self, name: str) -> typing.Union[dict, None]:
             return None
         return response['results'][0]
 
+    async def get_module_by_name_and_stream(
+        self,
+        name: str,
+        stream: str,
+    ) -> typing.Union[typing.List[dict], None]:
```

Comment:
We can use `typing.Optional` instead of `typing.Union`, because `typing.Optional[...]` is equivalent to `typing.Union[..., None]`

Reply:
Done